elixir/apps/domain/test/domain/config_test.exs

```diff
@@ -438,23 +438,6 @@ defmodule Domain.ConfigTest do
       %{account: account}
     end
 
-    test "returns error when changeset is invalid", %{account: account} do
```

Comment:
Same here Re: validations. Might be better to err on the conservative side and have the user contact us with their unsupported use case rather than mistakenly enter an invalid DNS host and wonder why queries are failing.